tenpy/linalg/symmetries.py

```diff
@@ -543,15 +542,9 @@ def __init__(self, factors: list[Symmetry]):
             braiding_style=max((f.braiding_style for f in flat_factors), key=lambda style: style.value),
             trivial_sector=np.concatenate([f.trivial_sector for f in flat_factors]),
             group_name=' ⨉ '.join(f.group_name for f in flat_factors),
-            num_sectors=np.prod([symm.num_sectors for symm in flat_factors]),
+            num_sectors= np.prod([symm.num_sectors for symm in flat_factors]), ####PROBLEM HERE WITH INT
```

Comment:
whats the problem?

Reply:
The problem is that in the common checks of  tests/linalg/test_symmetries.py it was only checked if num_sectors is of int type